app/controllers/v0/sessions_controller.rb

```diff
@@ -50,8 +50,9 @@ def user_attributes
         ssn:          attributes['social']&.first&.delete('-'),
         birth_date:   parse_date(attributes['birth_date']&.first),
         uuid:         attributes['uuid']&.first,
+        loa_highest:  attributes['level_of_assurance']&.first,
```

Comment:
just an idea but what about User.loa.current User.loa.highest then we can send in the profile { loa: { current: loa1, highest: loa3 } feels like a nice logical grouping